services/web/server/src/simcore_service_webserver/projects/_projects_service.py

```diff
@@ -1684,7 +1690,17 @@ async def _get_project_share_state(
                     ProjectStatus.CLONING,
                     ProjectStatus.EXPORTING,
                     ProjectStatus.MAINTAINING,
-                ],
+                ]
+                or (
+                    (
+                        app_settings.WEBSERVER_REALTIME_COLLABORATION.RTC_MAX_NUMBER_OF_USERS
```

Comment:
it reads so complicated...

THOUGTHS 
1. perhaps we should have used a very big number as default instead of using nullable trick ...
2. alternativlye we could have a helper within RealTimeCollaborationSettings ?

Reply:
well:
1. the ENV name is wrong
2. this is still under development